### PR TITLE
Contact

### PR DESCRIPTION
**Pull request by jtallen**
* Updated social media icons
* Updated Contact view copy
* Added email functionality to Contact form, added submission success message

### Before
Screen Shot 2022-01-13 at 4 06 35 PM

### After
Screen Shot 2022-01-13 at 4 06 41 PM

### Contact Form Success Confirmation
Screen Shot 2022-01-13 at 4 09 29 PM

### CODE REVIEW

File: components/Contact.js [hunk 1]
```diff
@@ -1,7 +1,83 @@
-import React from "react";
+import React, { useState } from "react";
 import userData from "@constants/data";
+import emailjs from "@emailjs/browser";
+
+function ContactForm(props) {
```
Comment (cchanningallen) [inline components/Contact.js]: Love the component extraction - nice work! 👌 

One suggestion: wdyta moving this to its own file? eg `ContactForm.js`?

The existing codebase idiom seems to be one-component-per-file, so seems reasonable to stay consistent.

File: components/Contact.js [hunk 2]
```diff
@@ -1,7 +1,83 @@
...
+        type="submit"
+        className="bg-blue-500 rounded-md w-1/2 mx-4 mt-8 py-2 text-gray-50 text-xs font-bold"
+      >
+        Send Message
+      </button>
+    </form>
+  );
+}
+
+function ContactFormSent() {
```
Comment (cchanningallen) [inline components/Contact.js]: Nice! Same suggestion re: extract to file

File: components/Contact.js [hunk 3]
```diff
@@ -1,7 +1,83 @@
...
+  return (
+    <div className="rounded-lg bg-white p-4 flex flex-col items-center py-24">
+      <h1 className="text-3xl font-semibold">Thanks for reaching out! 👋🏼</h1>
+      <p className="text-justify text-xl mx-4 my-16 border-t-2 border-b-2 border-[#02044A] p-2">
+        Unless I'm incapacitated 🤕 or travelling ✈️ you should hear from me
```
Comment (cchanningallen) [inline components/Contact.js]: 😆 love this. Slight tweak I might suggest:
```suggestion
        Unless I'm travelling ✈️ or ill 🤒 you should hear from me
```

File: components/Contact.js [hunk 4]
```diff
@@ -1,7 +1,83 @@
...
+    const message = e.target.elements.message.value;
+
+    emailjs
+      .send(userData.serviceID, userData.templateID, { name, email, message })
+      .then((res) => {
+        console.log("Email successfully sent!");
+        setIsSent(true);
+      })
+      .catch((err) => {
+        console.error("Oh no! Oh woe is me! The email has fail't!", err);
```
Comment (cchanningallen) [inline components/Contact.js]: 😆 love it. Do you want to maintain `console.log`s in your prod code? Totally cool if so, just double-checking

**Review by cchanningallen — COMMENTED** [review]
Looks great - nice work! So cool to see the interactivity. Component extraction was a great addition - makes the code way more legible.

A couple comments, lmkwyt.

File: components/Contact.js (hunk 4 above)
Comment (jtallen) [reply components/Contact.js]: Nope, good catch - thanks!

File: components/Contact.js (hunk 1 above)
Comment (jtallen) [reply components/Contact.js]: Good idea! Done! Noticed a couple places elsewhere in the code where this doesn't occur, so I will add it to the backlog to make this uniform.

File: components/ContactForm.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    >
+      <label htmlFor="name" className="text-sm text-gray-600 mx-4">
+        {" "}
+        Your Name
+      </label>
+      <input
+        type="text"
+        className="font-light rounded-md border focus:outline-none py-2 mt-2 px-1 mx-4 focus:ring-2 focus:border-none ring-blue-500"
+        name="name"
+      />
```
Comment (cchanningallen) [inline components/ContactForm.js:17]: Might be fun to extract this as an `Input` or `TextInput` component down the road, but out of scope for this PR I think

**Review by cchanningallen — APPROVED** [review]
Looks great! Awesome work 🙌 🎉

**Pull request merged**